### ISSUE TITLE
Tick strings on the bottom axis overlaps with its label

### ISSUE THREAD

**Issue by veractor** [domarm-comat/pglive#30]
Bottom axis tick strings do not seem to account for the drawing of axis labels. This could be recreated by changing line
```python
win = LivePlotWidget(title="Vertical Bar Plot @ 100Hz")
```

to:

```python
win = LivePlotWidget(title="Vertical Bar Plot @ 100Hz", labels={'bottom': ("hello"), 'left': ("hello")})
```
in `vertical_bar_plot.py` of examples. Here is what it looks like:

![x-axis overlap](https://github.com/domarm-comat/pglive/assets/56646254/376b2839-26d6-4652-892c-b2680bb84d29)

This does not seem to be a problem with other axes, just the bottom one.

**Comment by veractor** [comment]
I think this only happens with the latest release, v0.7.0.

**Comment by domarm-comat** [comment]
Thanks for debugging!
Yes, this was newly intorduced issue in 0.7.0. It should be fixed in 0.7.1.

**Comment by veractor** [comment]
Well, it sure doesn't overlap now, but there's more null space: 
![overshot](https://github.com/domarm-comat/pglive/assets/56646254/557f7ac6-02e8-4411-bce3-fa118e8c185f)

Compared to in v0.6.8:
![0 6 8](https://github.com/domarm-comat/pglive/assets/56646254/7b03aea2-60fb-4c98-b7e9-ea016eb1351d)

It's a minor nitpick though, as it fixes the problem. Ideally it would look exactly the same and the two axes would look more congruent, but it shouldn't matter that much.
